app/src/main/java/com/udacity/asteroidradar/api/NetworkHelper.kt

```diff
@@ -22,6 +20,7 @@ class NetworkHelper {
         private val client: OkHttpClient
         private val retrofit: Retrofit
         val retrofitService: NasaApiService
+        val retrofitServ: PictureOfDayInterface
```

Comment:
We should rename these to be different. Make the variable names closer to what class they are, someone from an outsiders perspective won't know the difference. You can right-click the val name and refactor it to something like

```
val nasaApiService: NasaApiService
val pictureOfDayService: PictureOfDayInerface
```